test/org/sagebionetworks/bridge/dynamodb/DynamoSchedulePlanDaoTest.java

```diff
@@ -71,6 +71,7 @@ public void canSerializeAndDeserializeSchedulePlan() throws Exception {
         String output = mapping.writeValueAsString(abPlan);
         
         SchedulePlan newPlan = mapping.readValue(output, SchedulePlan.class);
+        newPlan.setStudyKey(abPlan.getStudyKey()); // not serialized
         
         assertEquals("Schedule plans are equal", abPlan.hashCode(), newPlan.hashCode());
```

Comment:
Not sure why this doesn't just assertEquals(). Fixed both cases of this in test.